data/human/free worlds checkmate.txt

```diff
@@ -2333,25 +2356,213 @@ mission "FWC Pug 6"
+			choice
+				`	"We did what we had to."`
+				`	"We wouldn't have needed to use nukes of our own if you'd found the terrorists faster."`
+			`	Korban looks down, "Understandable," then looks back up at you with a smile, "but such is the cost of business, we say.`
+			label absolutely
+			`	"Now, we know that the terrorists still have nuclear weapons. We sent a ship to scout out their strength that barely made it out alive. We have a plan to take care of that, though.`
+			`	"Before the war, we began to work heavily toward developing automation for use on ships. You may have bought and used one of our Vanguards. Such a small crew needed to man a ship of such a size. We made some breakthroughs during the war that allowed us to drastically reduce the crew needed to pilot ships of any size, and we'll be sending a fleet of such ships as cannon fodder to have these terrorists waste their nukes."`
+			`	Freya looks shocked, "You're sending people on a suicide mission to take the nukes out of play? Can't we use diplomacy to stop adding bodies to the pile that this war has created?"`
+			`	Korban says, "These are terrorists, dear. They aren't to be negotiated with. But fear not, because these ships won't be manned. They'll be fully autonomous warships. Their combat skills are rough around the edges, and this will be an expensive loss - each of these ships costs us at least tens times the production of a normal ship - but much cheaper than the loss of innocent human lives."`
+			choice
+				`	"So where do we come into play?"`
+					goto play
+				`	"What are you going to do with this automation technology after this is all said and done?"`
+			`	"This technology is far from being profitable, but we hope to some day have the skies full of autonomous freighters transporting commodities across the galaxy faster than any merchant ever could, defended by deadly autonomous warships able to outsmart any pirate threat. That's neither here nor there, though. There are more important matters at hand.`
+				goto end
+			
+			label play
+			`	"We've already sent a single autonomous fleet to draw most of the terrorists' fire, but it wasn't enough. We have a second fleet at the ready, though, one for you to command. Being a skilled captain, I'm sure you'll be able to trick the terrorists into wasting whatever remains of their nukes before a proper fleet arrives to deal with them. We've contacted the Navy and told them where the terrorists are located, so they should be arriving shortly after you."`
+			
+			label end
+			`	Korban tells you that the second automamous fleet for you to lead is on Hephaestus. "The terrorists are holed up on Stormhold," says Korban, "an old pirate world. We've also made available to you some of our new technology, developed during the war. It may prove helpful to you during this final battle. Good luck, Captain."`
```

Comment:
```suggestion
			`	Korban tells you that the second autonomous fleet for you to lead is on Hephaestus. "The terrorists are holed up on Stormhold," says Korban, "an old pirate world. We've also made available to you some of our new technology, developed during the war. It may prove helpful to you during this final battle. Good luck, Captain."`
```